libethereum/ExtVM.cpp

```diff
@@ -33,7 +33,12 @@ namespace
 static unsigned const c_depthLimit = 1024;
 
 /// Upper bound of stack space needed by single CALL/CREATE execution. Set experimentally.
-static size_t const c_singleExecutionStackSize = 12 * 1024;
+static size_t const c_singleExecutionStackSize =
+#ifdef NDEBUG
```

Comment:
It should be set by cmake by default in all non-debug modes. The macro is a standard because the assert implementation depends on it.